README.rst

```diff
@@ -1,2 +1,54 @@
 This package provides an ExplainCollection class
-that allows common MongoDB commands to be explained
\ No newline at end of file
+that allows that allows PyMongo's Collection methods to be explained.
+
+
+Tutorial
+########
+The intended use case for this package is to debug ``pymongo`` commands.
```

Comment:
> ...to debug...

I am not sure that is correct. I think the use case is to ``explain`` operations run using PyMongo

Reply:
Done.